tools/spec-gen-sdk/src/automation/workflow.ts

```diff
@@ -160,8 +162,11 @@ export const workflowValidateSdkConfig = async (context: WorkflowContext) => {
   // only needs to check the two config provided case when both config enable the sdk generation or both config disable the sdk generation
   // the last case is the normal case, only one config enabled the sdk generation
   if (enabledSdkForTspConfig && enabledSdkForReadme) {
-    message = configError(`SDK generation configuration is enabled for both ${context.config.tspConfigPath} and ${context.config.readmePath}. Refer to https://aka.ms/azsdk/spec-gen-sdk-config to disable sdk configuration from one of them`);
-    throw new Error(message);
+    message = configWarning(`SDK generation configuration is enabled for both ${context.config.tspConfigPath} and ${context.config.readmePath}. ` +
+      `Refer to https://aka.ms/azsdk/spec-gen-sdk-config to disable sdk configuration from one of them. ` +
+      `This generation will be using ${context.config.tspConfigPath} to generate ${context.config.sdkName} SDK.`);
+    context.logger.warn(message);
```

Comment:
Do you also want to have a counter for this for the batch runs?

Reply:
I saved the result in the `execution-report.json` and it would be counted in the runner side. Can you review again?